apps/frontend/src/pages/access-management/libs/types/group-row.type.ts

```diff
@@ -3,6 +3,7 @@ type GroupRow = {
 	id: number;
 	name: string;
 	permissions: string[];
+	users: number;
```

Comment:
let's name it userCount for internal fields to make it more clear from the code, but the title for the column will remain "Users"
```suggestion
	userCount: number;
```